local/config/datavault.properties

```diff
@@ -28,7 +28,7 @@ db.password = datavault
 # Message queue settings
 # ======================
 # The location of the RabbitMQ server
-queue.server = rabbitmq
```

Comment:
Is that change good for all developers using this code? Will that break someone else's workflow?

Reply:
Yeah, this is fine. It's only us using the local config, and this change is to enable us to connect the broker running in Intellij to the rabbitmq running in docker so we can make deposits into vaults. It'll only break if someone has somehow mapped `http://localhost:15672` over to `http://rabitmq:15672` on their machine